demo/components/victory-area-demo.js

```diff
@@ -1,8 +1,8 @@
 /*global window:false */
 import React from "react";
 import { merge, random, range } from "lodash";
-import {VictoryArea, VictoryStack, VictoryGroup} from "../../src/index";
-import { VictoryContainer } from "victory-core";
+import {VictoryArea, VictoryStack, VictoryGroup, VictoryChart} from "../../src/index";
+import { VictoryContainer, Grayscale } from "victory-core";
```

Comment:
Let's give this a better name in victory-core. I think it should be namespaced under `Themes` or something.